### DISCUSSION TITLE
Draft links are suddenly broken on new project boards

### DISCUSSION THREAD

**Discussion by zeke** [community/community#16103] [Projects and Issues] [answered]
Hi 👋🏼 

I'm looking at https://github.com/orgs/replicate/projects/1/views/1

When I click the link below, it opens a new tab to https://github.com/orgs/replicate/projects/1/views/1# instead of opening a modal-ish thing for editing the draft.

Nothing in the Javascript console.

Confirmed with colleagues that this is happening on their computers today too.

<img width="355" alt="Screen Shot 2022-05-05 at 10 42 57 AM" src="https://user-images.githubusercontent.com/2289/166981801-ab153a47-0d51-4f6c-ae80-50e28dc31b1c.png">

**Comment by Fincap** [comment]
Also experiencing this.

**Comment by ch1bo** [comment]
Feedback items about this: #16083, #16093, #16103 (at least)

**Comment by eKristensen** [comment]
We also have this issue. Not nice.

**Comment by ch1bo** [comment]
Enabling the feature preview setting (again?) seems to be a solution to this! https://github.com/github/feedback/discussions/16136#discussioncomment-2699638

**Comment by azenMatt** [comment, marked as answer]
Hi - this is now fixed, but please let me know if it is still giving you problems
